styles.css

```diff
@@ -0,0 +1,20 @@
+#nav-bar {
+    position: fixed;
+    width: 100%;
+    top: 0;
+    left: 0;
+    font-family: 'Exo 2', sans-serif;
+    padding: 1em;
+    display: flex;
+    margin-top: 40px;
+  }
+  ul{
+    display: flex;
+    text-decoration: none;
+    list-style: none;
+    margin: 20px;
+  }
```

Comment:
Use this universal selector as a descendant of a specific class.i.e .nav-bar ul .inorder to be specific with that list or better give it a class.